doc/api/animation_api.rst

```diff
@@ -133,8 +133,37 @@ artist at a global scope and let Python sort things out.  For example ::
                        init_func=init, blit=True)
    plt.show()
 
-The second method is to use `functools.partial` to 'bind' artists to
-function.  A third method is to use closures to build up the required
+The second method is to use `functools.partial` to pass arguments to the
+function. ::
+
+   import numpy as np
+   import matplotlib.pyplot as plt
+   from matplotlib.animation import FuncAnimation
+   from functools import partial
+
+   fig, ax = plt.subplots()
+   line1, = ax.plot([], [], 'ro')
+
+   def init():
+       ax.set_xlim(0, 2*np.pi)
+       ax.set_ylim(-1, 1)
+       return line1,
+
+   def update(frame, ln, x, y):
+       x.append(frame)
+       y.append(np.sin(frame))
+       ln.set_data(x, y)
+       return ln,
+
+   xdata, ydata = [], []
```

Comment:
Thanks! Would be nice to have a way to detect these things automatically...

Reply:
You can try running `flake8-rst`, but unfortunately there are a lot of false positives since we don't always make code blocks self-contained.  It also seems to only work if you're explicitly using a directive like `.. code-block::`.